src/bot/commands/utilities/reverse.ts

```diff
@@ -0,0 +1,25 @@
+import { Command, Input, Listener } from '@api';
+
+export class Reverse extends Command {
+    constructor() {
+        super({
+            name: 'reverse',
+            description: 'Reverses the given text.',
+            arguments: [
+                {
+                    name: 'text',
+                    description: 'The text to reverse.',
+                    expand: true,
+                    required: true,
+                }
+            ]
+        });
+    }
+    
```

Comment:
To me the code seems like it's doing its reversing action in this chunk of the code, but I'm not really sure whether there's more code needed for this. I was going to choose to do the text reversing before but seemed like it would've taken too long to finish it, but if this is it then it seems like it's not too hard to make an attempt on it.